static/styles/zulip.css

```diff
@@ -3132,3 +3132,73 @@ div.topic_edit_spinner .loading_indicator_spinner {
         }
     }
 }
+
+#add-poll-modal {
+    height: 60vh;
+    overflow: hidden;
+
+    .simplebar-content {
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+
+        .modal__content {
+            flex-grow: 1;
+            overflow: hidden;
+        }
+    }
+}
+
+#add-poll-form {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+    height: 100%;
+
+    label {
+        font-weight: bold;
+        margin: 5px 0;
+    }
+
+    #poll-question-textarea {
+        /* Take up full width, leaving space for padding and border */
+        width: calc(100% - 14px);
+        box-shadow: none;
+        resize: none;
+        margin: 0;
+        margin-bottom: 10px;
+        height: 50px;
+    }
+
+    .poll-options-list {
+        margin: 0;
+        height: 0;
+        overflow: auto;
+        flex-grow: 1;
+    }
+
+    .option-row {
+        list-style-type: none;
+        cursor: move;
+        margin-top: 10px;
+        padding: 0;
+
+        .fa-ellipsis-v {
+            color: hsl(0, 0%, 75%);
+            position: relative;
+            top: 1px;
+
+            + i {
```

Comment:
What does the `+ i` do here? I'd expect our `fa-ellipsis-v` here to always be an `i` element...

Reply:
Actually, this is to add space after only the second (right side) icon to space it from the option text field. I'll separate it in a new block with a comment!